pkgs/by-name/js/json2cdn/package.nix

```
@@ -0,0 +1,57 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  makeWrapper,
+  gradle_8,
+  jre_headless,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "json2cdn";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "uku3lig";
+    repo = "json2cdn";
+    tag = finalAttrs.version;
+    hash = "sha256-pHmzeZHZyr3FyfkXwrdPk+lcHQKH6t4pnDD9ImMgSV8=";
+  };
+
+  nativeBuildInputs = [
+    gradle_8
+    makeWrapper
+  ];
+
+  mitmCache = gradle_8.fetchDeps {
+    inherit (finalAttrs) pname;
+    data = ./deps.json;
+  };
+
+  # this is required for using mitm-cache on Darwin
+  __darwinAllowLocalNetworking = true;
+
+  gradleBuildTask = "shadowJar";
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm644 build/libs/json2cdn-${finalAttrs.version}-all.jar $out/lib/json2cdn.jar
+
+    makeWrapper ${lib.getExe jre_headless} $out/bin/json2cdn --add-flags "-jar $out/lib/json2cdn.jar"
+
+    runHook postInstall
+  '';
+
```

```suggestion

  passthru = {
    tests.formats-cdn = tests.pkgs-lib.formats.passthru.entries.pass-cdnAtoms;
  };

```
It'd be best to add this as a package test (or alternatively, `tests.pkgs-lib`. Up to you)